### PR TITLE
Adding support for destination option

### PR DESCRIPTION
**Pull request by vilda**
Can you have a look @mrud ?

### CODE REVIEW

File: src/le.py [hunk 1]
```diff
@@ -199,10 +202,6 @@ def print_usage(version_only=False):
 #
 # Libraries
 #
-try:
-    import hashlib
-except ImportError:
-    pass
```
Comment (mrud) [inline src/le.py]: I think this was necessary for older versions of python for supporting needed algorithms in ssl. Can we leave that in?

**Comment by mrud** [comment]
besides the hashlib it looks good to me

File: src/le.py (hunk 1 above)
Comment (vilda) [inline src/le.py]: It's not used. Just importing this lib is enough to fix something?

Comment (StephenHynes7) [inline src/le.py]: @vilda http://bugs.python.org/issue8484 this is why it was put in even though it is not used.

Comment (vilda) [inline src/le.py]: Thanks, I'll put it back

**Comment by vilda** [comment]
@mrud import of hashlib is back now. You can merge this if it's ok.

**Pull request merged**